diff --git a/node_modules/next/dist/server/lib/router-utils/proxy-request.js b/node_modules/next/dist/server/lib/router-utils/proxy-request.js
index 74036e0..e32380f 100644
--- a/node_modules/next/dist/server/lib/router-utils/proxy-request.js
+++ b/node_modules/next/dist/server/lib/router-utils/proxy-request.js
@@ -27,6 +27,7 @@ async function proxyRequest(req, res, parsedUrl, upgradeHead, reqBody, proxyTime
         ignorePath: true,
         xfwd: true,
         ws: true,
+        secure: false,
         // we limit proxy requests to 30s by default, in development
         // we don't time out WebSocket requests to allow proxying
         proxyTimeout: proxyTimeout === null ? undefined : proxyTimeout || 30000
